lua/nvim-treesitter/config.lua

```diff
@@ -157,14 +157,15 @@ function M.norm_languages(languages, skip)
   end
 
   if not (skip and skip.dependencies) then
-    for _, lang in pairs(languages) do
+    for _, lang in ipairs(languages) do
       if parsers[lang] and parsers[lang].requires then
         vim.list_extend(languages, parsers[lang].requires)
       end
     end
   end
 
-  return languages
+  table.sort(languages)
+  return vim.fn.uniq(languages) ---@type string[]
```

Comment:
```suggestion
  return vim.fn.uniq(languages) --[=[@as string[]]=]
```
is I think more correct?